Desktop/analysis/analyses.cpp

```diff
@@ -339,8 +339,9 @@ void Analyses::rescanAnalysisEntriesOfDynamicModule(Modules::DynamicModule * mod
 			{
 				Analysis * a = keyval.second;
 
-				if(a->readyToCreateForm())
+				if(a->readyToCreateForm() && !a->justTranslated())
```

Comment:
I'm not sure to understand this fix... I see a risk that the form is not created at all...

Reply:
I dont really understand why the form would not be there?
Is `checkAnalysisEntry()` returning `false`?

Reply:
Its returning true.  therefore we reach the else statement